src/extension.ts

```diff
@@ -426,9 +426,9 @@ export class ExtensionManager implements vscode.Disposable {
                 // We have an active kit. We're good.
                 return true;
             }
-            // // No kit? Is enable kit scan on?
-            if (!this.workspaceConfig.enableKitScan) {
-                await cmakeProject.kitsController.setKitByName("");
+            // No kit? Is enable kit scan on?
+            if (!this.workspaceConfig.enableAutomaticKitScan) {
+                await cmakeProject.kitsController.setKitByName("__unspec__");
```

Comment:
I think we can actually reference `SpecialKits.Unspecified` here, that way we won't have to update multiple strings in the future if we ever change the unspecified string.